### PR TITLE
docs: correct a few broken links

### PR DESCRIPTION
**Pull request by csvn**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by JeanMeche — APPROVED** [review]
Looking good ! Thank you for helping us improve the docs !

File: adev/src/content/guide/components/dom-apis.md [hunk 1]
```diff
@@ -72,9 +72,9 @@ common use cases include:
 - Measuring element geometry, such as with `getBoundingClientRect`
 - Reading an element's text content
 - Setting up native observers such
-  as `[MutationObserver](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver)`,
-  `[ResizeObserver](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver)`, or
-  `[IntersectionObserver](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API)`.
+  as [MutationObserver](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver),
```
Comment (twerske) [inline adev/src/content/guide/components/dom-apis.md:75]: These should maintain the backtick inside the link name I believe

Comment (csvn) [reply adev/src/content/guide/components/dom-apis.md]: Fixed! I had trouble running Yarn and getting docs running locally, and did not see any other instances of code markdown inside links so I was not sure if that worked or not.

Comment (twerske) [reply adev/src/content/guide/components/dom-apis.md]: The docs don't yet run locally - we're still in the process of migrating the infrastructure to get them running locally.

**Review by twerske — APPROVED** [review]

**Comment by csvn** [comment]
Ah, sorry, I just tried syncing from the Gihub UI. Rebased and fixed it now!

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit cbd78af79d0a857f3c40086760b72700908eb6e9.

**Pull request closed without merging**